### ISSUE TITLE
Datepicker style not working on your starter

### ISSUE THREAD

**Issue by rakibhoossain** [element-plus/element-plus-vite-starter#38]
https://vite-starter.element-plus.org/
Datepicker scss style not working on your starter.

![alt text](https://i.ibb.co/mqd7nr8/Screenshot-2022-05-26-at-9-34-22-PM.png)

**Comment by YunYouJun** [comment]
Thanks for your feedback, I found the problem.

It is caused by the fact that the fixed namespace of the component is not dynamic. `el-picker-panel`
It still works just fine if you don't modify the namespace.
We will modify the dynamic class of element-plus as soon as possible.

**Comment by YunYouJun** [comment]
I create a PR https://github.com/element-plus/element-plus/pull/7911.

If you are interested, you can review it.

**Comment by rakibhoossain** [comment]
I've found some other namespace issues. I think there might be a lot of code written with static 'el' namespace.

1. When I hover on date field after selecting date, field size increasing.

I've solved it on my local project.
```css
.el-date-editor {
  display: inline-flex;
}
```

![field size increasing](https://i.ibb.co/CKNhd0x/Screenshot-2022-05-27-at-9-22-32-PM.png)


2. Margin missing on Dropdown button icon. It works fine on 'el' namespace. Because of static namespace.
I expect dynamic namespace here too.
```
.el-icon--right {
    margin-left: 5px;
}
```
![Margin missing on Dropdown button icon](https://i.ibb.co/QQvBYrF/Screenshot-2022-05-27-at-9-38-04-PM.png)
